### ISSUE TITLE
Project is not running on MacOs

### ISSUE THREAD

**Issue by potatmen** [objectionary/dejump#52]
In `RemoveGoto.java` there are some places like this:
```
final File dir = new File(
            String.format(
                "%s\\generated",
                this.path.substring(0, this.path.lastIndexOf('\\'))
            )
        );
```
That try to find path to directory using `\` slash, but on MacOs access to files is performed using `/`. After changing all backslash to front project works good.

**Comment by potatmen** [comment]
@MikhailLipanin

**Comment by 0pdd** [comment]
@potatmen the puzzle [#54](https://github.com/objectionary/dejump/issues/54) is still not solved.

**Comment by 0pdd** [comment]
@potatmen 3 puzzles [#54](https://github.com/objectionary/dejump/issues/54), [#57](https://github.com/objectionary/dejump/issues/57), [#58](https://github.com/objectionary/dejump/issues/58) are still not solved.

**Comment by 0pdd** [comment]
@potatmen 2 puzzles [#54](https://github.com/objectionary/dejump/issues/54), [#57](https://github.com/objectionary/dejump/issues/57) are still not solved; solved: [#58](https://github.com/objectionary/dejump/issues/58).

**Comment by 0pdd** [comment]
@potatmen 2 puzzles [#57](https://github.com/objectionary/dejump/issues/57), [#76](https://github.com/objectionary/dejump/issues/76) are still not solved; solved: [#54](https://github.com/objectionary/dejump/issues/54), [#58](https://github.com/objectionary/dejump/issues/58).
